snippets/rag-foundation/README.md

```diff
@@ -37,10 +37,19 @@ uv sync
 
 Make sure you have [Ollama](https://ollama.com/) installed and running locally. Pull the required model:
 
+```bash
+ollama serve
+```
```

Comment:
I think a check here might be good. However, different installation methods of ollama will result in different ways to start ollama. Maybe we send them to the [Ollama docs](https://github.com/ollama/ollama/tree/main/docs) if the ollama pull command fails.